### PR TITLE
Fix/GitHub ci workflows

### PR DESCRIPTION
**Pull request by JavanC**
- fix GitHub CI Workflows
- fix Maker integration test failed with LINK Vault debt ceiling

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: test
+name: Test
```
Comment (zodahu) [inline .github/workflows/test.yml:1]: For consistency, should we also modify the name in `deploy.yml`?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -8,12 +8,12 @@ on:
     branches: [master]
 
 jobs:
-  check:
+  test:
+    name: Run forge test
+    runs-on: ubuntu-latest
```
Comment (zodahu) [inline .github/workflows/test.yml:13]: For consistency, should we also modify the name and order in `deploy.yml`?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -28,11 +28,13 @@ jobs:
         run: |
           forge --version
           forge build --sizes
-        id: build
```
Comment (zodahu) [inline .github/workflows/test.yml:31]: For consistency, should we also remove all ids in `deploy.yml`?

File: .github/workflows/test.yml (hunk 1 above)
Comment (JavanC) [reply .github/workflows/test.yml:1]: Updated 🙏

File: .github/workflows/test.yml (hunk 2 above)
Comment (JavanC) [reply .github/workflows/test.yml:13]: Updated 🙏

File: test/integration/maker/AgentMakerAction.t.sol [hunk 4]
```diff
@@ -73,8 +73,8 @@ contract AgentMakerActionTest is Test, MakerCommonUtils, ERC20Permit2Utils {
         ethCdp = uint256(ret);
         assertEq(IERC20(DAI_TOKEN).balanceOf(user), DRAW_DAI_AMOUNT);
 
-        // Open LINK Vault
-        uint256 gemAmount = 500000 ether;
+        // Open WBTC Vault
```
[the remark this answers is not in the document]
Comment (JavanC) [reply test/integration/maker/AgentMakerAction.t.sol]: Updated 🙏

File: test/integration/maker/UtilityMaker.t.sol [hunk 5]
```diff
@@ -61,7 +61,7 @@ contract MakerUtilityTest is Test, MakerCommonUtils, ERC20Permit2Utils {
         IMakerVat vat = IMakerVat(VAT);
         bytes32 ilkETH = bytes32(bytes(ETH_JOIN_NAME));
         (, uint256 rate, uint256 spot, , uint256 dust) = vat.ilks(ilkETH);
-        (uint256 daiDrawMin, uint256 minCollateral) = _getDAIDrawMinAndMinCollateral(spot, dust);
+        (uint256 daiDrawMin, uint256 minCollateral) = _getDAIDrawMinAndMinCollateral(spot, dust, 18);
```
Comment (hihiben) [inline test/integration/maker/UtilityMaker.t.sol:64]: Since the last parameter is the decimal of collateral, is it the same as the GEM_DECIMAL?

Comment (JavanC) [reply test/integration/maker/UtilityMaker.t.sol:64]: It's collateral decimal, and in here is the LockETH case so is 18 (GEM_DECIMAL  for LockGem)

Comment (hihiben) [reply test/integration/maker/UtilityMaker.t.sol:64]: I see. In that case I think defining the constant just like GEM_DECIMAL will be better for understanding.

Comment (JavanC) [reply test/integration/maker/UtilityMaker.t.sol:64]: like ETH_DECIMAL for LockETH case~?

Comment (JavanC) [reply test/integration/maker/UtilityMaker.t.sol]: Updated!

**Review by hihiben — APPROVED** [review]
LGTM

**Pull request merged**